scripts/corner-math.js

```diff
@@ -9,15 +9,9 @@ export function offset_for_curvature(curvature) {
   // curve that has the same point at t = 0.5.
   if (curvature <= 0.001)
     return [1, -1];
-  const {
-    x
-  } = se(curvature);
-  const [a, b] = [x, 1 - x].map(m => 2 * m - 0.5);
-  const magnitude = Math.hypot(a, b);
-  // Normalize a & b
-  const norm_a = a / magnitude;
-  const norm_b = b / magnitude;
-  return [norm_a, -norm_b];
+  const s = Math.min(1, Math.max(-1, Math.log2(curvature)));
+  const rotation = (1 - s) * Math.PI / 4;
+  return [Math.cos(rotation), -Math.sin(rotation)]
```

Comment:
Is this better than what I have?
```
      const x = this.superellipsePointAtProgress(s, 0.5).x;
      const offset = 2 - Math.sqrt(2); // Chosen so that with s=1 (k = 2) this returns { 1, 0 } which is necessary to match historical `corner-shape: round` behavior.
      const [a, b] = [x, 1 - x].map(m => 2 * m - offset);
      const result = new Size(Math.max(a, 0), Math.max(b, 0));
```

Reply:
I am actually not sure what's better in terms of explaining it in the spec.
Both produce very similar numbers.

- The previous formula is the perpendicular to the hull of the superellipse (control point of the approximate arc)
- The new one is rotating the stroke.